contrib/storage-http/src/test/java/org/apache/drill/exec/store/http/TestHttpPlugin.java

```diff
@@ -585,7 +585,7 @@ public void testSlowResponse() throws Exception {
       server.enqueue(
         new MockResponse().setResponseCode(200)
           .setBody(TEST_JSON_RESPONSE)
-          .throttleBody(64, 4, TimeUnit.SECONDS)
```

Comment:
@vdiravka 
Thanks for the PR.  For this test, we could also set the timeout for the plugin config to be say 1 seconds and set the throttle to be 2.  Just a suggestion.  The goal with this test was just to make sure the plugin does in fact time out and throw the appropriate exception.

Reply:
I made this [change](https://github.com/vdiravka/drill/commit/55e5731270fa87b86a3a3c493ae3c969614568be#diff-c6b42ccd4a5372e7c299b3b8052455c0fe618403a957e0b1b908edf46eee4809R151), but then test sometimes can fail:
```
2021-08-12T22:02:14.1585593Z [ERROR] Tests run: 23, Failures: 1, Errors: 0, Skipped: 4, Time elapsed: 9.967 s <<< FAILURE! - in org.apache.drill.exec.store.http.TestHttpPlugin
2021-08-12T22:02:14.1588729Z [ERROR] org.apache.drill.exec.store.http.TestHttpPlugin.testSlowResponse  Time elapsed: 2.047 s  <<< FAILURE!
2021-08-12T22:02:14.1590406Z java.lang.AssertionError
2021-08-12T22:02:14.1592236Z 	at org.apache.drill.exec.store.http.TestHttpPlugin.testSlowResponse(TestHttpPlugin.java:597)
2021-08-12T22:02:14.1593656Z 
2021-08-12T22:02:14.2122033Z [DEBUG] Closing the fork 1 after saying GoodBye.
2021-08-12T22:02:14.2128185Z [INFO] 
2021-08-12T22:02:14.2128842Z [INFO] Results:
2021-08-12T22:02:14.2129375Z [INFO] 
2021-08-12T22:02:14.2129898Z [ERROR] Failures: 
2021-08-12T22:02:14.2130733Z [ERROR]   TestHttpPlugin.testSlowResponse:597
2021-08-12T22:02:14.2131508Z [INFO] 
2021-08-12T22:02:14.2132131Z [ERROR] Tests run: 35, Failures: 1, Errors: 0, Skipped: 
```
So looks like this functionality works not precisely. I will check whether it can be improved or it is allowable error (~2-3sec)

Reply:
@vdiravka, I was just making a suggestion there.  Feel free to modify whatever you need to in order to get the CI to work consistently.  I also do concur with @dzamo that we should increase the timeout from 100m if we can because it seems like tests will randomly timeout.

Reply:
@cgivre, if tests are timed out, it is because they hang, not because the timeout is low. Usually, tests are completed in 60 min.